### PR TITLE
uploading /Second_Batch/Pratiksha/myInfo.md

### PR DESCRIPTION
**Pull request by pratikshaghule**

### CODE REVIEW

File: Second_Batch/Pratiksha/myInfo.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+I am pratiksha.....
```
Comment (shankar7791) [inline Second_Batch/Pratiksha/myInfo.md]: please add more information about you....

File: Second_Batch/Pratiksha/myInfo.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+branch line: uploading into branch pratiksha_work
```
Comment (shankar7791) [inline Second_Batch/Pratiksha/myInfo.md:3]: add your city

**Review by shankar7791 — APPROVED** [review]
you can merge it

File: Second_Batch/Pratiksha/myInfo.md (hunk 2 above)
Comment (coolyasu) [reply Second_Batch/Pratiksha/myInfo.md:3]: add more details

**Pull request merged**